<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.149, y1:0.130364, x2:0.93, y2:0.942727, stop:0 rgba(0, 255, 158, 255), stop:0.9801 rgba(255, 255, 255, 255));
}
</string>
   </property>
   <widget class="QLabel" name="Label_Nazov">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;; color:Black;</string>
    </property>
    <property name="text">
     <string>Afinni Sifra</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_A">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hodnota A</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_B">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hodnota B</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheckBox_Sifrovat">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sifrovat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheckBox_Desifrovat">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Desifrovat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Execute">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Vykonaj</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_A">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_B">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVysledek">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>1181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>VÃ½sledek = </string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
